governance/program/src/instruction.rs

```diff
@@ -523,6 +523,12 @@ pub enum GovernanceInstruction {
     ///   1. `[writable]` ProposalDeposit account. PDA seeds: ['proposal-deposit', proposal, deposit payer]
     ///   2. `[writable]` Proposal deposit payer (beneficiary) account
     RefundProposalDeposit {},
+
+    /// Verify the state and move the Proposal to Completed.
+    /// Permission-less operation whoever could call.
+    ///
+    ///   0. `[writable]` Proposal account.
+    CompleteProposal {},
```

Comment:
Nit: can you add more comments about when this is used? For example, specifying "this instruction only works for `Succeeded` proposals that have no instructions".

And perhaps the name can be changed to `CompleteEmptySuccessfulProposal` or some other name that describes the intention more.

You're probably more aware of the terms used in the space, so I'll defer to you!

Reply:
@joncinque thanks. I elaborated a bit more on function of the instruction in comment.
I tend to use the generic `CompleteProposal` name but I would like to ask @SebastianBor  for opinion on this.
This instruction should  (or could) have two parts. This PR resolves only one of them. I created a separate issue to make it clear (my wrong not doing so before).
The one that this PR resolves is
https://github.com/solana-labs/solana-program-library/issues/3772
The other (a potentially future PR) is:
https://github.com/solana-labs/solana-program-library/issues/4047

Reply:
Ah that's great, thanks for clarifying!  Since all of the other comments are in the enum description, could you move your comments on `complete_proposal` up to here? And if it were me, I'd rename it to `CompleteStuckProposal`, but I'll let you figure that out :smile:

Reply:
Thank you @joncinque .

I fixed the place where the documentation comment resides at + I renamed the instruction to `CompleteStuckProposal` as you suggested.
If you find time to check it would be great.